core/src/epicli/data/common/defaults/configuration/applications.yml

```diff
@@ -66,11 +66,11 @@ specification:
 ## --- Pgpool ---
 
   - name: pgpool # this service requires PostgreSQL to be installed in cluster
-    enabled: no
+    enabled: yes
```

Comment:
Just wondering of by default we dont want to disable this like the rest of the application deployments? What if someone does not install postgress?

Reply:
Fixed, thx